css/compositing/parsing/background-blend-mode-computed-multiple.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Compositing and Blending Level 1: getComputedStyle().backgroundBlendMode</title>
+<link rel="help" href="https://drafts.fxtf.org/compositing-1/#propdef-background-blend-mode">
+<meta name="assert" content="background-blend-mode computed value is as specified even when the number of images vary.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/computed-testcommon.js"></script>
+<style>
+    #target {
+        background-image: none, none, none;
```

Comment:
Might be better to specify something that isn't `none` just in case (`linear-gradient(green, green), linear-gradient(green, green), ...` or so.

But this is fine too.

Reply:
fixed.